api/definition/swagger.yaml

```diff
@@ -0,0 +1,273 @@
+openapi: 3.0.0
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/ANDREWJEVANS/fingertips-next-api/1.0.0
```

Comment:
Does this server reference need to be in here? Looks like it's used to define the server to communicate with for the "Try it out" functionality in generated documentation, but presumably we don't want to be pointing it at your instance of a Swaggerhub server?